src/chapters/how-programs-work/programming-languages.rst

```diff
@@ -10,7 +10,7 @@ Given a set of inputs, Han's computer analyzes the data and returns information
 about safely navigating a hyperspace jump. The computer does this by running a
 **program**.
 
-At the most basic level, a *program* is a set of instructions that tell a
+At the most basic level, a *program* is the code needed to follow a set of instructions that tell a
```

Comment:
Since the book does not define "code" until the next chapter, I'm not sure we should use the term here.  Thoughts?

Reply:
I was a little confused about why "code" didn't have a formal definition until later since much of the first chapter talks about learning to code so I went ahead and used it in this sentence. I am totally fine with removing it! I do think that this definition of "program" is confusing because algorithms are established as a set of instructions. I added something there to make it clear that the program is the actual written instruction in code. Maybe "file" works better? But then many programs contain multiple files. Thoughts?

Reply:
I see your point.  How about if we replace "set of instructions" with "series of algorithms" (or "set of...", or "collection of...")?

Reply:
To that point, I think there should be a way to effectively state that a program is the *code*. An algorithm could be written down on a piece of paper. What about "A *program* is how we communicate to the computer what we need it to do to follow an algorithm."?